mirrord/config/src/outgoing/unix.rs

```diff
@@ -0,0 +1,147 @@
+use mirrord_config_derive::MirrordConfig;
+use schemars::JsonSchema;
+
+use crate::{
+    config::{from_env::FromEnv, source::MirrordConfigSource, Result},
+    util::{MirrordToggleableConfig, VecOrSingle},
+};
+
+// TODO: update docs.
+/// Specify which paths of tcp unix sockets should be connected to on the remote target and which
+/// should be connected to locally.
+/// Paths can be specified as regular expressions according to
+/// [this syntax](https://docs.rs/regex/1.7.1/regex/index.html#syntax).
+///
+/// If you only want to specify a single path (or regular expression), you don't need to put it
+/// inside an array, you can pass the string directly.
+///
+/// You can either specify the remote paths (allow list) or the local paths (block list), not both.
+/// If you specify the remote paths, only the specified paths will be connected to remotely, and
+/// all other paths will be connected to on your local system.
+/// If you specify the local paths, all paths will be connected to remotely, except for the
+/// specified paths.
+///
+/// ## Examples
+///
+/// ### Connect to all paths remotely by including them all with a regular expression
+///
+/// ```toml
+/// # mirrord-config.toml
+///
+/// [feature.network.outgoing.unix_tcp]
+/// remote = ".*"
+/// ```
+///
+/// ### Connect to all paths remotely by passing an empty local list
+///
+/// ```toml
+/// # mirrord-config.toml
+///
+/// [feature.network.outgoing.unix_tcp]
+/// local = [] # Don't do anything locally - connect to all unix addresses remotely.
+/// ```
+///
+/// ### Connect to all paths locally by completely disabling the unix_tcp option
+///
+/// ```toml
+/// # mirrord-config.toml
+///
+/// [feature.network.outgoing]
+/// unix_tcp = false
+/// ```
+///
+/// ### Connect to all paths remotely except for paths in /tmp/local-socket and paths that end with "-local"
+///
+/// ```toml
+/// # mirrord-config.toml
+///
+/// [feature.network.outgoing.unix_tcp]
+/// local = [
+///     "^/tmp/local-socket/.*",
+///     "-local$",
+/// ]
+/// ```
+#[derive(MirrordConfig, Default, PartialEq, Eq, Clone, Debug)]
+#[config(map_to = "UnixTcpFileConfig", derive = "JsonSchema")]
+#[cfg_attr(test, config(derive = "PartialEq, Eq"))]
+pub struct UnixTcpConfig {
+    /// Connect to these paths remotely (and to all other paths locally).
+    ///
+    /// Value is a list separated by ";".
+    #[config(env = "MIRRORD_OUTGOING_UNIX_TCP_REMOTE_LIST")]
```

Comment:
It's a `VecOrSingle` also in the file configuration.
https://github.com/metalbear-co/mirrord/blob/780845a5af7d435f98d20cdb3e816def22ea2a10/mirrord/config/src/fs/advanced.rs#L64

The 

> list separated by ";"

 part is copied from the env config 
 https://github.com/metalbear-co/mirrord/blob/780845a5af7d435f98d20cdb3e816def22ea2a10/mirrord/config/src/env.rs#L57-L59
  
  and should probably be deleted both here and there, since it's shown in the documentation for the configuration file, I think.